libkbfs/disk_block_cache.go

```diff
@@ -96,11 +96,12 @@ type MeterStatus struct {
 }
 
 func rateMeterToStatus(m metrics.Meter) MeterStatus {
+	s := m.Snapshot()
 	return MeterStatus{
-		m.Rate1(),
-		m.Rate5(),
-		m.Rate15(),
-		m.RateMean(),
+		s.Rate1(),
+		s.Rate5(),
+		s.Rate15(),
+		s.RateMean(),
```

Comment:
Maybe we should change `MeterStatus.Mean` to `MeterStatus.Count`, and use `s.Count()` here?